cmd/cmd/localEntryShow.go

```diff
@@ -53,7 +52,7 @@ func init() {
 	localEntryCmd.AddCommand(localEntryShowCmd)
 	localEntryShowCmd.SetHelpTemplate(showLocalEntryCmdLongDesc +
 		utils.GetCmdUsage(programName, localEntryCmdLiteral,
-		utils.ShowCommand, "[localentry-name]") + showLocalEntryCmdExamples +
+			utils.ShowCommand, "[localentry-name]") + showLocalEntryCmdExamples +
```

Comment:
Let's correct formatting :)

Reply:
I believe the formatting is correct as I ran [`go fmt`](https://blog.golang.org/go-fmt-your-code) on the project.